bucklescript/src/TableclothString.ml

```diff
@@ -0,0 +1,161 @@
+type t = string
```

Comment:
Thats right, to avoid shadowing the String which is sued in the implementation.

If there is I couldn't find it, in Janestreets codebase they create a separate `Caml` module but that wouldn't work for the Bucklescript side (no equivalent do multiple `dune` files)